docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-alerts-health-check.md

```diff
@@ -59,6 +61,167 @@ Regardless of the alert level, you create an alert as follows:
 
 - Click **Save**.
 
+### Templates reference
+
+Alert templates available in YugabyteDB Anywhere have been created using Prometheus expressions. Although not required, you might be inclined to consult the following list of expressions:
+
+- ```expression
+  expr: count by (node_prefix) (yb_node_ysql_write_read{node_prefix=""} < 1)
```

Comment:
Shouldn't we state explicitly - which expression corresponds to which alert template?
Sometimes it may be hard to say looking at just the expression itself.

Reply:
@anmalysh-yb As I noted in Jira and here in the PR description, it is not clear why platform users would need to know these expressions, considering that alert templates are created by the platform dev and the user's task is to make a selection from the list fed into the Templates combo box.

It would take me quite a bit of time to create a map of expressions and their corresponding templates available through the UI.  In addition, as you pointed out, this list is going to change all the time and maintaining it would be problematic. 

Until we have a clear external use case for this info, the doc team's effort doesn't seem fully justified.

Reply:
The use case is actually clear.
Some customers will want to use their in-house monitoring solution to view Yugabyte DB metrics and control alerts.
They will federate metrics from our embedded Prometheus instance to their own metric system in this case.
But to configure alerts - they will need to know - which expression do we use for our alerts - in order to prepare similar expressions for their system

Reply:
Okay, thanks.